docker/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Start the REST API with 4 workers
+echo "Starting REST mode..."
+gunicorn "keep.api.api:get_app(mode='rest')" \
+  --bind 0.0.0.0:8080 \
+  --workers 4 \
+  -k uvicorn.workers.UvicornWorker \
+  -c /venv/lib/python3.11/site-packages/keep/api/config.py &
+
+# Check if REDIS is true, and start the worker mode if it is
+# I'm using random port 1234 because it doesn't matter
+if [ "$REDIS" = "true" ]; then
```

Comment:
I don't think REDIS could be definitive for worker, shouldn't api be able to schedule tasks for worker via redis?

Reply:
Other words, API should be able to access REDIS